src/main/java/com/microalgo/controller/StringController.java

```diff
@@ -0,0 +1,19 @@
+package com.microalgo.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.microalgo.constants.AlgoServiceConstants.URIConstants.STRING_PING_END_POINT;
+
+@RestController
+public class StringController
+{
+    @GetMapping(STRING_PING_END_POINT)
+    public ResponseEntity<String> pingStringController()
+    {
+        ResponseEntity<String> pingResponse=new ResponseEntity<String>(HttpStatus.OK);
```

Comment:
Do we need a mention of datatype within angle brackets on right hand side ? Cant we just use the angle brackets only.

Reply:
ok, removed. I dont know what the standard is around this, I guess one of those things down to personal choice :)